solidity/contracts/utility/Utils.sol

```diff
@@ -53,4 +53,15 @@ contract Utils {
     function _validExternalAddress(address _address) internal view {
         require(_address != address(0) && _address != address(this), "ERR_INVALID_EXTERNAL_ADDRESS");
     }
+
+    // ensures that the fee is valid
+    modifier validFee(uint32 fee) {
```

Comment:
Utils is a generic contract with no bancor specific logic. This is a deviation from that path.
Can this function be defined such as that it's generic?

Reply:
It is somewhat generic (fee is not a Bancor-specific term after all), but a better direction can be to introduce something like `TokenUtils` and move this and some other shared methods (such as a ``NATIVE_TOKEN_ADDRESS` aware transfer of balance retrieval methods) there.

Reply:
We can also just treat it as bancor utils, practically that's what it is I guess, better than having 2 separate contracts

Reply:
👍